### ISSUE TITLE
Add Logout button

### ISSUE THREAD

**Issue by hypermystic** [hypermystic/bookit#13]
Button should be available in the bottom.
After logging off it will redirect to the Sign in page

**Comment by binismail** [comment]
Hey @hypermystic,

can I work on this issue?

**Comment by hypermystic** [comment]
yeah sure...assigned and if you face any problem in setting up the project..please read the instructions in issue "auto redirect to https://todo-7e79c.web.app/"
